controllers/operators/operators_objects.go

```diff
@@ -0,0 +1,290 @@
+package operators
+
+import (
+	sv1b2 "github.com/RedHatInsights/strimzi-client-go/apis/kafka.strimzi.io/v1beta2"
+	lv1beta1 "github.com/ViaQ/loki-operator/api/v1beta1" //TODO replace ViaQ repo by grafana/loki/operator one when released
+	gv1alpha1 "github.com/grafana-operator/grafana-operator/v4/api/integreatly/v1alpha1"
+	"github.com/netobserv/network-observability-operator/api/v1alpha1"
+	"github.com/netobserv/network-observability-operator/controllers/constants"
+	"github.com/netobserv/network-observability-operator/controllers/reconcilers"
+	ov1 "github.com/operator-framework/api/pkg/operators/v1"
+	ov1alpha1 "github.com/operator-framework/api/pkg/operators/v1alpha1"
+	pv1 "github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func operatorGroup(name string, namespace string) *ov1.OperatorGroup {
+	return &ov1.OperatorGroup{
+		TypeMeta: metav1.TypeMeta{
+			Kind:       constants.OperatorGroup,
+			APIVersion: "operators.coreos.com/v1",
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      name,
+			Namespace: namespace,
+			Labels:    map[string]string{"app": constants.ObservabilityName},
```

Comment:
here and everywhere else the `constants.ObservabilityName` is used, I think `app` isn't the best label to use, maybe "part-of" would be better? cf https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/#labels except we remove the `app.kubernetes.io/` prefix for shorter labels

Reply:
(or `managed-by`)